app/components/paginator/view.rb

```diff
@@ -0,0 +1,79 @@
+# frozen_string_literal: true
+
+module Paginator
+  class View < ViewComponent::Base
+    attr_reader :scope
+
+    # This constant limits the number of links to pages rendered by #paginate.
+    # In some cases, the number of links will be KAMINARI_LINKS_LIMIT + 1 because
+    # kaminari refuses to leave a gap of just one number e.g. 1 ... 345 ... 9
+    # so it renders 1 2 3 4 5 ... 9 instead.
+    KAMINARI_LINKS_LIMIT = 5
+
+    def initialize(scope:)
+      @scope = scope
+    end
+
+    def render?
+      @scope.total_pages > 1
```

Comment:
Feeling daft... 

since there is  `attr_reader :scope`

shouldn't we drop the `@`
```suggestion
      scope.total_pages > 1
```

Reply:
you ain't daft!